design-editor/src/panel/preview/preview-element.js

```diff
@@ -165,14 +168,34 @@ class Preview extends DressElement {
 	createPreviewDocument(contents, location) {
 		const relativePathToFile = pathUtils.joinPaths(location, 'temporary-preview.html'),
 			writeFile = promisify(fs.writeFile),
-			parsedContents = removeMediaQueryConstraints(contents);
+			parsedContents = this.addLibrary(removeMediaQueryConstraints(contents));
 		this.fsPath = pathUtils.createProjectPath(relativePathToFile);
 
 		return writeFile(this.fsPath, parsedContents)
 			.then(() => {
 				return pathUtils.createProjectPath(relativePathToFile, true);
 			}).catch((err) => {throw err;});
 	}
+
+	addLibrary(contents) {
+		const library = this.libraryCreator.createLibrary('preview-helper.css'),
+			domParser = new DOMParser(),
+			xmlSerializer = new XMLSerializer(),
+			doc = domParser.parseFromString(contents, 'text/html');
+
+		doc.head.appendChild(
+			library.createHTMLElement(
+				checkGlobalContext('globalData').fileUrl
+			)
+		);
+
+		library.insertLibContent(() => {
+			// eslint-disable-next-line no-console
+			console.log('OK Library copied');
```

Comment:
do we really need this console log?
if yes let it will be informative e.g. "preview-helper.css library copied"

Reply:
Done!